app/services/reports/usps_report.rb

```diff
@@ -0,0 +1,42 @@
+require 'login_gov/hostdata'
+
+module Reports
+  class UspsReport < BaseReport
+    REPORT_NAME = 'usps-report'.freeze
+
+    def initialize
+      @results = {}
+      @results[:today] = Time.zone.today.to_s
+      @results[:letters_sent_since_days] = {}
+      @results[:letters_sent_and_validated_since_days] = {}
+      @results[:percent_sent_and_validated_since_days] = {}
```

Comment:
Nitpick, please disregard if feels over-correct. IMO express better the intention of 'initialize the @results ivar with default values':
```suggestion
      @results = {
        today: Time.zone.today.to_s,
        letters_sent_since_days: {},
        letters_sent_and_validated_since_days: {},
        percent_sent_and_validated_since_days: {}
      }
```